build.fsx

```diff
@@ -241,16 +241,22 @@ Target "CodeGen.NetCore" (fun _ ->
     let open2 = "--open Microsoft.FSharp.Compiler"
     let open3 = "--open Microsoft.FSharp.Compiler"
 
-    // dotnet restore
-    run "dotnet" "restore -v Information"
+    // restore all the required tools, declared in each fsproj
+    !! "**/*netcore.fsproj"
+    ++ "tests/service.netcore/service.fsproj"
+    |> Seq.iter (fun path ->
+        let parent = System.IO.FileInfo(path).DirectoryName
+        tracefn "restoring at %s" path
+        runCmdIn parent "dotnet" "restore %s -v detailed" path
```

Comment:
if you add the `.sln`, you can `dotnet restore src/FSharp.Compiler.netcore.sln`
directory doesnt matter if you Fullname the `path` of fsproj/sln
It's a lot faster to use `sln` because nuget is slow